examples/example_plugin_with_view_override/example_plugin_with_view_override/ui/CustomView.jsx

```diff
@@ -1,30 +1,35 @@
-import Table from 'react-bootstrap/Table';
+import { Table, Box, TableContainer, Tbody, Thead, Tr, Th, Td } from '@nautobot/nautobot-ui';
```

Comment:
Are the changes in `example_plugin_with_view_override` unrelated to this PR?

Reply:
Yup unrelated, Came across them, and decided to fix them

Reply:
Aren't these in conflict with the changes being made in #3721? I don't understand how these two PRs relate to one another.